compiler/rustc_query_system/src/dep_graph/dep_node.rs

```diff
@@ -96,15 +95,6 @@ pub struct DepNode {
     pub hash: PackedFingerprint,
 }
 
-// We keep a lot of `DepNode`s in memory during compilation. It's not
-// required that their size stay the same, but we don't want to change
-// it inadvertently. This assert just ensures we're aware of any change.
-#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
-static_assert_size!(DepNode, 18);
-
```

Comment:
This comment should be good information to keep.

Reply:
The comment on the added `size_asserts` module says "Some types are used a lot. Make sure they don't unintentionally get bigger" which is basically the same information, and the same comment used on all of these size assertion modules. So I think that's equivalent and good enough.